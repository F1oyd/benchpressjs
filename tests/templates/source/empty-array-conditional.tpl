<!-- IF emptyarray.length -->
array is not empty
<!-- ELSE -->
array is empty
<!-- END -->

<!-- IF emptyarray -->
array is not empty
<!-- ELSE -->
array is empty
<!-- END -->