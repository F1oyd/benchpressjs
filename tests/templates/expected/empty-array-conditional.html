array is empty

array is empty